app/assets/javascripts/hydra-editor/field_manager.es6

```diff
@@ -122,7 +122,7 @@ export class FieldManager {
 
     getFieldLabel($element, options) {
         var label = '';
-        var $label = $element.find("label");
+        var $label = $element.find("label").first();
```

Comment:
@hackmastera you're test appeared to cover the scenario I was questioning, so I think we're good.